src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Typography } from '../Typography';
+import { DivStyled } from './Badge.styles';
+import { BadgeProps } from './types';
+
+const Badge: React.FC<BadgeProps> = ({
+    text,
+    state = 'normal',
+    textVariant = 'body16',
```

Comment:
sort alphabetically for @BillyG83 's eyes 👀

Reply:
Done.

Reply:
there is a VSC plugin called sortLine @AbhinavMV TY dude!